core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -129,41 +131,10 @@ case class Submission(
   useCallCache: Boolean,
   deleteIntermediateOutputFiles: Boolean,
   workflowFailureMode: Option[WorkflowFailureMode] = None,
-  cost: Option[Float] = None
+  cost: Option[Float] = None,
+  externalEntityInfo: Option[ExternalEntityInfo] = None
 )
 
-case class SubmissionStatusResponse(
-  submissionId: String,
-  submissionDate: DateTime,
-  submitter: String,
-  methodConfigurationNamespace: String,
-  methodConfigurationName: String,
-  submissionEntity: Option[AttributeEntityReference],
-  workflows: Seq[Workflow],
-  status: SubmissionStatus,
-  useCallCache: Boolean,
-  deleteIntermediateOutputFiles: Boolean,
-  workflowFailureMode: Option[WorkflowFailureMode] = None,
-  cost: Option[Float] = None
-)
-object SubmissionStatusResponse {
-  def apply(submission: Submission): SubmissionStatusResponse =
-    SubmissionStatusResponse(
-      submissionId = submission.submissionId,
-      submissionDate = submission.submissionDate,
-      submitter = submission.submitter.value,
-      methodConfigurationNamespace = submission.methodConfigurationNamespace,
-      methodConfigurationName = submission.methodConfigurationName,
-      submissionEntity = submission.submissionEntity,
-      workflows = submission.workflows,
-      status = submission.status,
-      useCallCache = submission.useCallCache,
-      deleteIntermediateOutputFiles = submission.deleteIntermediateOutputFiles,
-      workflowFailureMode = submission.workflowFailureMode,
-      cost = submission.cost
-    )
-}
```

Comment:
this differed from `Submission` only in the type of `submitter`. This was probably more significant in the pre-sam era but now it looks like this class was just extra junk.